contrib/!config/dockerfile/README.org

```diff
@@ -0,0 +1,26 @@
+#+TITLE: Dockerfile contribution layer for Spacemacs
+
+[[file:img/docker.png]]
+
+* Table of Contents                                                   :TOC@4:
+ - [[#description][Description]]
```

Comment:
These links do not work for me in emacs. Clicking on it asks if I want to make a new heading with the title.